### DISCUSSION TITLE
启动doris  be 报错

### DISCUSSION THREAD

**Discussion by yclxiao** [apache/doris#14071] [A - General / Q&A]
启动doris  be 报错
<img width="975" alt="image" src="https://user-images.githubusercontent.com/8273732/200502617-db5d598d-e9e2-47d1-be03-66bce0ec4859.png">

**Comment by zy-kkk** [comment]
请问是哪个版本

**Reply by AlvisZhao** [reply]
1.1.2 版本be也报了这个错，在这之前，BE出现过OOM

**Reply by zy-kkk** [reply]
可以升级1.1.3版本，应该修复过类似的问题
